test/unit/deferred.js

```diff
@@ -525,6 +525,34 @@ QUnit.test( "jQuery.Deferred.then - spec compatibility", function( assert ) {
 	} catch ( _ ) {}
 } );
 
+QUnit[ window.console ? "test" : "skip" ]( "jQuery.Deferred.exceptionHook", function( assert ) {
```

Comment:
`testIf` has already been contemplated [multiple](https://github.com/jquery/qunit/issues/637#issuecomment-53498233) [times](https://github.com/jquery/qunit/pull/714#issuecomment-67778915), and is easy to add manually. Let's try it out in jQuery core. :+1: